src/_sass/home2.scss

```diff
@@ -352,8 +454,70 @@
     row-gap: $spacing-unit;
   }
 }
-
 .Home2-resources-listItem {
   border: 1px solid $tilt2-color-gray-lightest;
-  padding: $spacing-unit;
+  padding: $spacing-unit / 2;
+  box-shadow: 0px 0px ($spacing-unit / 2) 0px rgba(180, 180, 180, $tilt2-alpha-transparent);
+  border-radius: $tilt2-border-radius;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+.Home2-resources-listItem-text {
+  margin-bottom: $spacing-unit / 4;
+}
+
+.Home2-resources-icon {
+  margin-right: $spacing-unit / 2;
+}
+
+.Home2-resources-button {
+  @include reset-button-style;
+  line-height: $tilt2-line-height-body;
+  width: 100%;
+}
+.Home2-resources-link,
+.Home2-resources-button {
+  display: block;
+  font-family: $tilt2-font-sans;
+  font-weight: $tilt2-font-weight-sans-bold;
+  font-size: $tilt2-font-size-body;
+  line-height: $tilt2-line-height-body;
+  padding-top: $spacing-unit / 4;
+  padding-bottom: $spacing-unit / 4;
+  border-radius: $tilt2-border-radius;
+  border: 1px solid $color-blue;
+  text-align: center;
+  color: $color-blue;
+
+  &:hover {
+    color: $color-white;
```

Comment:
should this have a `transition` on color/background-color?

Reply:
Yes! Good note.